interfaces/html.idl

```diff
@@ -1783,7 +1783,7 @@ interface Window : EventTarget {
 
   // the user agent
   readonly attribute Navigator navigator;
-  readonly attribute Navigator clientInformation; // legacy alias of .navigator
+  [Replaceable] readonly attribute Navigator clientInformation; // legacy alias of .navigator
```

Comment:
This change makes Firefox fail with 'setter must be function for PutForwards, Replaceable, or non-readonly attributes expected "function" but got "undefined"'.

That's the only test regression it seems, and expected.